js/service/shell/index.js

```diff
@@ -44,3 +44,20 @@ exports.setCommand = function(name, cb) {
   assert(typeutils.isFunction(cb));
   commands.set(name, cb);
 };
+
+exports.runCommand = function(name, args, cb) {
+  var capture = require('./captureio.js');
```

Comment:
Oh, sorry. The higlighted part included the function definition. Sorry, misunderstood you.